src/main/kotlin/com/chutneytesting/kotlin/dsl/ChutneyScenarioDsl.kt

```diff
@@ -585,9 +152,9 @@ class ChutneyStep(
 
 class ChutneyStepImpl(
     val type: String,
-    @JsonInclude(NON_EMPTY) val target: String?,
-    @JsonInclude(NON_EMPTY) val inputs: Map<String, Any?>,
-    @JsonInclude(NON_EMPTY) val outputs: Map<String, Any>?
+    @JsonInclude(NON_EMPTY) val target: String? = null,
```

Comment:
Isn't target property a mandatory variable ? or there are tasks where it's not ?

Reply:
Yes like debug task.